pkg/reader/ccip.go

```diff
@@ -628,6 +628,10 @@ func (r *ccipChainReader) discoverDestinationContracts(
 	}
 	for chain, cfg := range sourceConfigs {
 		resp = resp.Append(consts.ContractNameOnRamp, chain, cfg.OnRamp)
+		// The local router is located in each source chain config. Add it once.
+		if len(resp[consts.ContractNameOnRamp][r.destChain]) == 0 {
+			resp = resp.Append(consts.ContractNameRouter, r.destChain, cfg.Router)
```

Comment:
Discussed offline: explain that we want to discover the wrapped native address and that both router and testrouter should point to the same wrapped native, and that this is a system-level assumption that is not explicitly checked offchain.